projects/dev/src/app/group/group-content/group-content.component.ts

```diff
@@ -26,14 +27,7 @@ export class GroupContentComponent implements OnInit {
       tab.classList.remove('mat-tab-label-before-active');
     });
     
-    let i;
-
-    for (i = 0 ; i < tabs.length ; i++) {
-      if (tabs[i] === activeTab) {
-        break;
-      }
-    }
-
+    const i = _.findIndex(tabs, activeTab);
```

Comment:
Yes, but it wasn't working as `tabs` is not Array type.
So I used lodash instead.
But if you want, I can do it in a Javascript way.
`tabs` is NodeList so convert it to Array type and use `indexOf` function instead of `_.findIndex`.

Reply:
Lodash is ok, I was just surprised as you said you would convert to native js.